scripts/coretemp/download-language-packs.cmd

```diff
@@ -0,0 +1 @@
+powershell %~dp0download-language-packs.ps1
```

Comment:
I think just the ps1 file should be sufficient, no? We don't usually have Cmd scripts in the repo

Reply:
We could update the note to `Download language packs by running: $dir\\download-language-packs in PowerShell`

Reply:
How about this:
`Run $dir\download-language-packs.ps1 (powershell $dir\download-language-packs.ps1 under cmd)`

Reply:
Yup, seems good